plugin.js

```diff
@@ -159,6 +168,12 @@ function casPlugin (server, options) {
       username: session.username,
       attributes: session.attributes
     }
+    if (_options.value.sessionCredentialsMappings) {
+      const dotProp = require('dot-prop')
+      for (let mapping of _options.value.sessionCredentialsMappings) {
+        dotProp.set(credentials, mapping.credentialsAttribute, dotProp.get(session, mapping.sessionAttribute))
```

Comment:
There's a lot of work going on here for something that is really simple:

https://github.com/jsumners/nixconfig/blob/master/lib/getPathValue.js
https://github.com/jsumners/nixconfig/blob/37987334d80ddff24a2b815a0747ffd4b7093f4d/lib/nixconfig.js#L91-L103

Do we really need the extra two dependencies (1 direct & 1 indirect) for this?